src/components/sq-modal-profile-cache/sq-modal-profile-cache.component.tsx

```
@@ -0,0 +1,244 @@
+
+  const resetState = useCallback(() => {
+    const newState = fieldsCache?.reduce(
+      (acc, field) => ({
+        ...acc,
+        [field]: new Intl.NumberFormat('pt-br', { minimumFractionDigits: 2 }).format(
+          socialNetworkObject?.socialNetworkProfilesCache?.find(({ contentType }) => contentType === field)?.contentValue ?? 0,
+        ),
+      }),
+      {},
+    ) as Record<CacheField, string>
+    onStateChange && onStateChange(newState)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)